tidb-cloud/integrate-tidbcloud-with-n8n.md

```diff
@@ -187,18 +189,99 @@ return [{json: {response}}];
 9. Set **Message** field mode to `Expression`
 10. Enter `{{ $json["response"] }}` in the **Message**.
 
-> Note: It is very important to mouse over the Message field and select the Expression pattern.
+> **Note:** 
+> 
+> It is very important to mouse over the Message field and select the Expression pattern.
```

Comment:
hover

Reply:
@zhangyangyu Thank you for your careful review. I have used `hover` instead of `over`.